dist/src/main/java/io/camunda/zeebe/broker/BrokerModuleConfiguration.java

```diff
@@ -70,7 +71,8 @@ public BrokerModuleConfiguration(
       final BrokerShutdownHelper shutdownHelper,
       final PrometheusMeterRegistry meterRegistry,
       final SecurityConfiguration securityConfiguration,
-      final UserServices userServices) {
+      // The UserServices class is not available if you want to start-up the Standalone Broker
+      final Optional<UserServices> userServices) {
```

Comment:
🔧 ❓ Could we solve this with an overloaded constructor instead of an optional parameter?

Reply:
We could, but is it more clear then using Optional ?

We have another option:

```
@Autowired
public BrokerModuleConfiguration(
    final BrokerBasedConfiguration configuration,
    final IdentityConfiguration identityConfiguration,
    final SpringBrokerBridge springBrokerBridge,
    final ActorScheduler actorScheduler,
    final AtomixCluster cluster,
    final BrokerClient brokerClient,
    final BrokerShutdownHelper shutdownHelper,
    final PrometheusMeterRegistry meterRegistry,
    final SecurityConfiguration securityConfiguration,
    @Autowired(required = false) final UserServices userServices) {
  this.configuration = configuration;
  this.identityConfiguration = identityConfiguration;
  this.springBrokerBridge = springBrokerBridge;
  this.actorScheduler = actorScheduler;
  this.cluster = cluster;
  this.brokerClient = brokerClient;
  this.shutdownHelper = shutdownHelper;
  this.meterRegistry = meterRegistry;
  this.securityConfiguration = securityConfiguration;
  this.userServices = userServices;
}
```

I don't have a strong preference